justfile

```diff
@@ -28,6 +38,14 @@ test-testproject:
     cd testproject && cargo odra test -b casper
     cd testproject && cargo odra clean
 
+test-workspace-project:
+    cd testproject && rustup target add wasm32-unknown-unknown
+    cd testproject && rustup component add rustfmt --toolchain nightly-2023-03-01-x86_64-unknown-linux-gnu
```

Comment:
Is this required to add the toolchain explicitly? Isn't it defined in the `rust-toolchain` file?